runelite-client/src/main/java/net/runelite/client/plugins/xptracker/XpInfoBox.java

```diff
@@ -197,7 +199,6 @@
 	void reset()
 	{
 		canvasItem.setText(ADD_STATE);
-		container.remove(statsPanel);
```

Comment:
Why is this line removed?

Reply:
`statsPanel` was moved to be inside `headerPanel` at some point, so this line had no effect.